source/includes/swift/_objects.md

```diff
@@ -0,0 +1,262 @@
+`q` <br/>*string* | Filters the object with the name contains the value of the parameters.
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the object .
+`name` <br/>*string* | The name of the object.
+`displayName`<br/>*string* | The name rendered for display.
+`path`<br/>*string* | The path of the object.
+`size`<br/>*int* | The size in bytes of the object. It is 0 for object that are used as folders (Generated by UI).
+`isNode`<br/>*boolean* | The access type of the container. It can either be public or private.
+`bucketName`<br/>*string* | The name of the bucket that the object is part of.
+`displaySize`<br/>*object* | Object containing the information to display the size in the UI.
+`contentType`<br/>*string* | The mime type corresponding to the content of the file.
+`lastModified`<br/>*string* | The last date that the file was modified.
+
+<!-------------------- RETRIEVE AN OBJECT -------------------->
+
+#### Retrieve an object
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/swift/example/objects/exampleFolder/sampleFile.txt?bucket_name=private"
+
+# The above command returns JSON structured like this:
+```
+
+```json
+{
+  "id": "private/exampleFolder/sampleFile.txt",
+  "name": "sampleFile.txt",
+  "displayName": "sampleFile.txt",
+  "path": "",
+  "size": 27,
+  "isNode": false,
+  "bucketName": "private",
+  "displaySize": {
+    "value": "27",
+    "unitKey": "units.filesizes.B"
+  },
+  "contentType": "application/octet-stream",
+  "lastModified": "Wed Mar 11 16:51:20 EDT 2020"
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/objects/:path_without_container?bucket_name=:bucket_name</code>
+
+Retrieve an object in a given [environment](#administration-environments) in a specified container.
+
+Mandatory | &nbsp;
+------- | -----------
+`bucket_name` <br/>*string* | The name of the bucket from which the object must be fetched. 
+
+
+Attributes | &nbsp;
+------- | -----------
+`id` <br/>*string* | The id of the object .
+`name` <br/>*string* | The name of the object.
+`displayName`<br/>*string* | The name rendered for display.
+`path`<br/>*string* | The path of the object.
+`size`<br/>*int* | The size in bytes of the object. It is 0 for object that are used as folders (Generated by UI).
+`isNode`<br/>*boolean* | The access type of the container. It can either be public or private.
+`bucketName`<br/>*string* | The name of the bucket that the object is part of.
+`displaySize`<br/>*object* | Object containing the information to display the size in the UI.
+`contentType`<br/>*string* | The mime type corresponding to the content of the file.
+`lastModified`<br/>*string* | The last date that the file was modified.
+
+
+<!-------------------- DOWNLOAD AN OBJECT -------------------->
+
+#### Download an object (To complete)
```

Comment:
`(To complete)` Should we merge something that isn't implemented yet?

Reply:
good point. @simongodard what is your take on this?

Reply:
We should only merge completed doc. I would remove that section if not completed.

Reply:
Should i keep only the container since the objects needs some fixes?

Reply:
Yes

Reply:
will do

Reply:
d1a1327e888bdf3d2f16d91831f2c29d5a3e36f6